patches/tmvTable.patch

```diff
@@ -1,12 +1,13 @@
-diff --git a/ups/tmv.table b/ups/tmv_LSST.table
-index 006ea09..b9f4d57 100644
---- a/ups/tmv_LSST.table
+diff --git a/ups/tmv.table b/ups/tmv.table
+index 006ea09..b4ec4c5 100644
+--- a/ups/tmv.table
 +++ b/ups/tmv.table
-@@ -1,5 +1,4 @@
+@@ -1,5 +1,5 @@
 +setupRequired(scons)
  envPrepend(PATH,${PRODUCT_DIR}/bin)
  envPrepend(LD_LIBRARY_PATH,${PRODUCT_DIR}/lib)
 -envPrepend(LIBRARY_PATH,${PRODUCT_DIR}/lib)
 -envPrepend(C_INCLUDE_PATH,${PRODUCT_DIR}/include)
 -envPrepend(CPATH,${PRODUCT_DIR}/include)
 +envPrepend(DYLD_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
++envPrepend(LSST_LIBRARY_PATH, ${PRODUCT_DIR}/lib)
```

Comment:
Can you fix the spacing here (remove the space) so that after the eupspkg prep phase there is not a diff against the git version of the table file.